### PR TITLE
Feature social auth

### PR DESCRIPTION
**Pull request by kamyanskiy**
prototype deployed here - http://lit-everglades-9568.herokuapp.com/

### CODE REVIEW

**Comment by kamyanskiy** [comment]
@tatij review me please

File: cookbook/decorators.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from functools import wraps
+
+from django.template import RequestContext
+from django.shortcuts import render_to_response
+
+
+def render_to(tpl):
```
Comment (oleg-chubin) [inline cookbook/decorators.py]: we can try something from django annoying

**Comment by oleg-chubin** [comment]
Is it possible to use bootstrap from external resource and do not have it in our git repo?

or maybe we can improve deployment procedure and download bootstrap  when needed.

This is minor issue though

**Comment by kamyanskiy** [comment]
@oleg-chubin I removed a lot of static bootstrap files (still the only 2, we need them to display login form and starter bootstrap theme). Now we use main bootstrap css and js from outside.
The second, I splitted templates, now all of them use base.html and I move navbar to external block "navbar".

**Comment by kamyanskiy** [comment]
I added django-annoying lib and removed my bicycle.

**Comment by kamyanskiy** [comment]
deployed here http://lit-everglades-9568.herokuapp.com/

**Comment by oleg-chubin** [comment]
Чувак, мегокруто!!!

Таня посмотрит, что получилось и если замечаний вопросов не будет вливаемся :)

**Pull request merged**